.github/workflows/docker-image.yml

```diff
@@ -16,6 +16,24 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v2
+      # Dependency Caching Steps
+      # Standard Java Workflow Steps
+      - uses: actions/checkout@v1
+      - name: Setup JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      # Cache Save and Restore
+      - name: Java Dependency Cache Step
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      # Run Maven Tests
```

Comment:
These tests are not well maintained. I don't think we should be running them as part of our CI at this point.
If you remove this step, does the build pass?

Reply:
I didn't have them prior and it worked. When I was doing some knowledge digging, I saw people adding `mvn test` as a best practice. If it's unstable, I understand keeping it out. I just reran the pipeline without the tests though and there seems to be a generic issue. The steps for setting up JDK and and caching do pass. This error is at the `Set up QEMU` step. 

```Pulling binfmt Docker image
  /usr/bin/docker pull tonistiigi/binfmt:latest
  Error response from daemon: Get "https://registry-1.docker.io/v2/": EOF
  Error: The process '/usr/bin/docker' failed with exit code 1

Reply:
Yeah, makes sense. It is a best practice to run unit tests, provided we have a working unit test suite. I expect those tests to fail, but we could always investigate and see. I think the error you posted (JChem dependency not found) is expected if we run `mvn test` with no other arguments, because it'll try to use the default JChem build profile. But you might be able to get further by passing the `-P ${CHEMISTRY_PACKAGE}` argument to specify either the bbchem or indigo flavors.
See the Dockerfile which runs `mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}`


That generic issue looked like a transient issue with access to some public resource, so I reran your jobs and they seem to have passed this time: https://github.com/mcneilco/acas-roo-server/runs/8076853823?check_suite_focus=true

Reply:
@dalejerikson I was curious so I re-ran those jobs another time, to see if they would be faster. Unfortunately they didn't seem to go much faster, and the logging indicates that it was re-downloading maven dependencies.
I tried to verify using the GitHub Actions Cache API (https://docs.github.com/en/rest/actions/cache#list-github-actions-caches-for-a-repository)
But https://api.github.com/repos/mcneilco/acas-roo-server/actions/cache/usage indicates there are no caches for this repo.
Could you dig in further and try to get a GitHub actions run where it is clearly using the cache and going faster?

Reply:
<img width="1019" alt="Screen Shot 2022-08-30 at 9 55 06 AM" src="https://user-images.githubusercontent.com/97194463/187455787-8e95b5c6-b6a5-48d1-bd75-0255379556c7.png">
Hmmmm wonder if this, at least in part, is the cause.

Reply:
Interesting. You could try adding a pull request trigger to this workflow, at least temporarily. That should make it so that you can push commits to this PR and it'll re-trigger builds.